src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let men;
+
+  century
+    ? men = people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
+    : men = people.filter(person => person.sex === 'm');
```

Comment:
I think you can make this assignment more beautiful, 
`men = century
   ?...
   :...`

Reply:
Yep you rigth. Nice propose)